taf/updater/updater.py

```diff
@@ -58,38 +59,49 @@ def update():
   loads data from a most recent commit.
   """
 
-  # temporary, during initial development
-  clients_directory = 'E:\\OLL\\tuf_updater_test'
-  repository_name = 'dc-law'
+  # TODO old HEAD as an input parameter
 
-  clients_reposiotry = os.path.join(clients_directory, repository_name)
-  clients_keystore = os.path.join(clients_directory, 'keystore')
-  clients_metadata = os.path.join(clients_reposiotry, 'metadata')
+  clients_repository = os.path.join(clients_directory, repo_name)
 
   # Setting 'tuf.settings.repository_directory' with the temporary client
   # directory copied from the original repository files.
   tuf.settings.repositories_directory = clients_directory
```

Comment:
`clients_repository` from line 64. is not used. Did you mean `tuf.settings.repositories_directory = clients_repository`?

Reply:
I think that so, but it doesn't seem to make any difference.

Reply:
It did make a difference. This is resolved though.